src/style.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-weight: 300;
+  --gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search_form_big {
+  margin: 20px auto;
+}
+
+.search_bar {
```

Comment:
Right now font weight on input (not placeholder) is wrong, it's 300 now, but should be 900